### ISSUE TITLE
Default Merge All Fields Option on Object Merge Handler

### ISSUE THREAD

**Issue by CowboyDan** [kyleschmid/ObjectMerge#4]
Provide a checkbox on the Object Merge Handler called "Default Merge All Fields". 

Instead of relying on Object Merge Field records, all fields would be merged.

This would remove the need to create any Object Merge Field records, with the exception of:

1. Object Merge Fields marked as "Use for Matching"

1. Object Merge Fields marked as "Preserve Null Values" (https://github.com/kyleschmid/ObjectMerge/issues/3)

**Comment by kyleschmid** [comment]
Would want the ability to specify certain fields to ignore with a checkbox on Object Merge Field called "Skip Field"

**Comment by CowboyDan** [comment]
Doesn't "Preserve Null" net out to the same as a skip? If the Master is null, it won't change, and if it had a value, it wouldn't change.

**Comment by kyleschmid** [comment]
You're right. Didn't think all the way through that one. That comment is a duplicate of #3. Also might want to make it a little more robust with checkboxes, since they are true or false, not null.
